.github/workflows/build.yml

```diff
@@ -45,18 +41,23 @@ jobs:
       - name: Check out code and set up JDK and Maven
         uses: s4u/setup-maven-action@4f7fb9d9675e899ca81c6161dadbba0189a4ebb1 # v1.18.0
         with:
-          java-version: ${{ matrix.jdk }}
-          java-distribution: ${{ matrix.distribution }}
+          java-version: |
+            ${{ matrix.target-jdk }}
+            ${{ matrix.jdk }}
```

Comment:
I seem to recall that this syntax doesn't work with `s4u/setup-maven-action`; to be investigated.

Reply:
Indeed, I already filed an issue for this back in January: s4u/setup-maven-action#112. Will try to work around it.